hildon/hildon-entry.c

```diff
@@ -67,11 +67,15 @@ set_property                                    (GObject      *object,
     switch (prop_id)
     {
     case PROP_SIZE:
-	size = g_value_get_flags (value);
-	/* If this is auto height, default to finger height. */
-	if (!(size & (HILDON_SIZE_FINGER_HEIGHT | HILDON_SIZE_THUMB_HEIGHT)))
-	  size |= HILDON_SIZE_FINGER_HEIGHT;
-	hildon_gtk_widget_set_theme_size (GTK_WIDGET (object), size);
+        size = g_value_get_flags (value);
+        /* If this is auto height, default to finger height. */
+        if (!(size & (HILDON_SIZE_FINGER_HEIGHT | HILDON_SIZE_THUMB_HEIGHT)))
+            size |= HILDON_SIZE_FINGER_HEIGHT;
```

Comment:
Please, move such style-fixing changes to a separate commit to distinguish them from code changes.
In this example commit message can be: "hildon/hildon-entry.c: fix tabs/spaces mess"
If several files are changed, just omit the filename part.

Reply:
This are minor and time-consuming changes. Do we really have to complicate our lives that much for such minor changes?

Reply:
Nice commit history is really appreciated if someone wants to hack on the code.
"git blame" shouldn't confuse users.

I understand why you don't want to work on splitting/rebasing commits, I did it myself several times.

It's better to do now. And this is really important thing in such complicated project which Maemo Leste is.

BTW, "sanitizing all trailing whitespaces", you mentioned above, could go to the same commit.